examples/allennlp/allennlp_jsonnet.py

```diff
@@ -0,0 +1,63 @@
+"""
+Optuna example that optimizes a classifier configuration for IMDB movie review dataset.
+This script is based on the example of allentune (https://github.com/allenai/allentune).
+
+In this example, we optimize the validation accuracy of
+sentiment classification using AllenNLP cli.
+Since it is too time-consuming to use the training dataset,
+we here use the validation dataset instead.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python allennlp_cli.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize allennlp_cli.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+import os.path
+import shutil
+
+import optuna
+from optuna.integration import AllenNLPExecutor
+
+
+# This path trick is used since this example is also
+# run from the root of this repository by CI
+EXAMPLE_DIR = os.path.dirname(os.path.abspath(__file__))
+MODEL_DIR = "result"
```

Comment:
@toshihikoyanase Let @himkt and me know should you have any comments here.

The original discussion is here: https://github.com/optuna/optuna/pull/1086#discussion_r405213926

Reply:
SGTM. Thanks!